src/common/components/dropdownSelect/__tests__/DropdownSelect.test.tsx

```diff
@@ -0,0 +1,184 @@
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+import DropdownSelect, {
+  dataTestIds,
+  DropdownSelectProps,
+} from '../DropdownSelect';
+
+const defaultIds = {
+  getItemId: (index: number) => `test_id-item-${index}`,
+};
+
+const renderSelect = (props: DropdownSelectProps) => {
+  const ui = <DropdownSelect {...props} />;
+
+  const wrapper = render(ui);
+  const label = wrapper.getByText(props.labelText || '');
+  const menu = wrapper.getByRole('listbox');
+  const toggleButton = wrapper.getByTestId(dataTestIds.toggleButton);
+
+  const getItemAtIndex = (index: number) =>
+    wrapper.getByTestId(dataTestIds.item(index));
+
+  const getItems = () => wrapper.queryAllByRole('option');
```

Comment:
I think this should be used instead of getItemAtIndex

screen.getAllByRole('option')[options.length - 2] => getItems()[options.length - 2]